src/util/misc.ts

```diff
@@ -347,3 +347,11 @@ export function objToQueryString(obj: KVS<any>): string {
 export function sanitizeString(str: string) {
   return str.replace(/[&<>"'*=//]/g, ' ');
 }
+
+export function getArticleBase(articleId: string) {
```

Comment:
I would like to see here a comment explaining what does this function exactly and probably a better name as well.

Reply:
The function name is changed to encodeArticleId(), which seems more explanatory and mentioned a comment which explains this function only seems to encode "?" and "#" characters.